GOVERNANCE.md

```diff
@@ -194,6 +194,6 @@ If needed, we reserve the right to publicly announce removal.
 
 [coc]: https://github.com/grafana/grafana/blob/master/CODE_OF_CONDUCT.md
 [devs]: https://groups.google.com/forum/#!forum/grafana-developers
-[maintainers.md]: https://github.com/search?l=&q=org%3Agrafana+filename%3AMAINTAINERS.md+path%3A%2F&ref=advsearch&type=Code&utf8=%E2%9C%93
+[maintainers]: https://github.com/grafana/grafana/blob/master/MAINTAINERS.md
```

Comment:
Shouldn't we add a maintainers.md to the other repositories as well? and then the old link would still be valid?